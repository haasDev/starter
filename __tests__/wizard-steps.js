import React from 'react';
import WizardSteps from '../src/share/wizard-steps';
import snapshot from './snapshot';

describe('WizardSteps', () => {
  it('should render', () => {
    const props = {
      steps: [
        'Assign Products',
        'Add Trail Data',
        'Add Observations'
      ]
    };

    snapshot(<WizardSteps {...props} />);
  });
});
