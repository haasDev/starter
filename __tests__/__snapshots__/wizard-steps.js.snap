exports[`WizardSteps should render 1`] = `
<div
  className="wizard-steps-container"
  style={
    Object {
      "display": "flex",
      "justifyContent": "space-around",
    }
  }>
  <div
    className="step-container">
    <span
      className="badge">
      1
    </span>
     
    Assign Products
  </div>
  <div
    className="step-container">
    <span
      className="badge">
      2
    </span>
     
    Add Trail Data
  </div>
  <div
    className="step-container">
    <span
      className="badge">
      3
    </span>
     
    Add Observations
  </div>
</div>
`;
